### DISCUSSION TITLE
How to customize the physical space key on Kindle 3 for the "Next Page" page turn button ?

### DISCUSSION THREAD

**Discussion by zhanglin2022** [koreader/koreader#12321] [Q&A] [answered]
How to customize the physical space key on Kindle 3 for the "Next Page" page turn button ? Because it's much more convenient to press the space key than the "Next Page" page turn buttons of two sides in landscape mode(ratate 180 degrees) and I use the landscape mode very often.

**Comment by Commodore64user** [comment]
Update to latest nightly OR wait for the next stable release 2024.08

**Reply by zhanglin2022** [reply]
ok

**Reply by Commodore64user** [reply, marked as answer]
for some clarity, this is already in the code base. So if you want it _today_, download the nightly.

**Reply by zhanglin2022** [reply]
Thanks for reminding me of this.

**Reply by zhanglin2022** [reply]
May I ask how to set it up specifically? I cannot find the option to set the `space key` as the `page down key` in the menu "setting-Navigation-physical buttons".

**Reply by Commodore64user** [reply]
You can’t. It’s hardcoded to always go forward
